config/routes.rb

```diff
@@ -111,6 +108,19 @@
     end
   end
 
+  constraints Clearance::Constraints::SignedIn.new(&:admin?) do
+    namespace :settings, module: nil do
+      scope :signal, module: :settings do
+        get '/', to: 'signal#edit', as: 'signal_server_phone_number'
+        patch '/', to: 'signal#update'
+        get '/register', to: 'signal#captcha_form'
+        post '/register', to: 'signal#register'
+        get '/verify', to: 'signal#verify_form'
+        post '/verify', to: 'signal#verify'
+      end
+    end
+  end
```

Comment:
Wow! I lost track that we include that feature.

Reply:
Actually, I removed the controllers/views/components for this workflow, but forgot to remove the routes. Thanks for catching this.